ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -160,6 +172,10 @@ VirtualizationRootHandle VirtualizationRoot_FindForVnode(
         }
         
         vnode_t parent = vnode_getparent(vnode);
+        if (nullptr == parent)
```

Comment:
Minor: Should we be comparing against `NULLVP` rather than `nullptr`? (I am assuming they have the same value, but I've seen us use `NULLVP` elsewhere).

Reply:
Yep, good catch. And yes, they're essentially the same, except with a bit of extra type safety.